modules/storage/minio.go

```diff
@@ -90,6 +95,19 @@ func NewMinioStorage(ctx context.Context, cfg *setting.Storage) (ObjectStorage,
 		return nil, convertMinioErr(err)
 	}
 
+	// Check if the connection works
+	err = getBucketVersioning(ctx, minioClient, config.Bucket)
```

Comment:
Why not invoke `_, err := minioClient.GetBucketVersioning(ctx, config.Bucket)` directly here?